lib/src/executable.dart

```diff
@@ -47,7 +47,7 @@ main(List<String> args) async {
 
   var color = (options['color'] as bool) ?? hasTerminal;
   try {
-    var css = render(options.rest.first, color: color);
+    var css = await renderAsync(options.rest.first, color: color);
```

Comment:
Defaulting to the current isolate's package resolver only makes sense here if Sass is being used as a library or a direct dependency of a package. If it's being run as a global or standalone executable—which is better supported anyway—that logic won't work.

Probably the best thing to do here is to follow Dart's behavior for auto-detecting package resolution from the Sass file's location, and allow users to override it with a `--packages` flag. For now, just remove the default behavior and implement the flag; we can do the auto-detection in a separate CL.